bool      safe_to_go
bool      safe_to_operate
bool      unplugging
bool      ac_power
int16     laptop_battery_percent
bool      dangerous_doors_open